pkg/autoupdate/tuf/autoupdate_test.go

```diff
@@ -116,7 +116,7 @@ func TestExecute_launcherUpdate(t *testing.T) {
 	shutdownDeadline := time.Now().Add(5 * time.Second).Unix()
 	for {
 		if time.Now().Unix() > shutdownDeadline {
-			t.Error("autoupdater did not shut down within 5 seconds")
+			t.Error("autoupdater did not shut down within 5 seconds -- logs: ", logBytes.String())
```

Comment:
I wonder if this is going to be readable, or kind of a mess. I wonder about just copying it to `os.Stderr`

Reply:
It'll definitely be a little messy, but it's hopefully not permanent! I don't think we'd see output if we copy to `os.Stderr` -- I find that output from tests tends to get swallowed in CI, and if I write something to `os.Stderr` in a test I don't see that output when running `make test` locally.